### PR TITLE
Rename d2l-quick-eval-activities-list to d2l-quick-eval-submissions-table

### PR DESCRIPTION
**Pull request by AlexBedley**
I renamed:

* The element itself
* All the files
* All the labels (such as the test suite titles and the demo headers)
* All the css classes/ids
* All event names

I did NOT rename:
* The variables named `list`. I'm thinking about doing it in a future PR however

### CODE REVIEW

File: components/d2l-quick-eval/d2l-quick-eval-submissions.js [hunk 1]
```diff
@@ -350,28 +350,12 @@ class D2LQuickEvalSubmissions extends mixinBehaviors(
...
 			return result.then(sortedCollection => {
 				this.entity = sortedCollection;
-				this._dispatchSortUpdatedEvent(sortedCollection);
 			});
 		} else {
 			return Promise.reject(new Error(`Could not find sortable header for ${headerId}`));
 		}
 	}
 
-	_dispatchSortUpdatedEvent(sorted) {
```
Comment (AlexBedley) [inline components/d2l-quick-eval/d2l-quick-eval-submissions.js]: I accidentally committed this (I meant to make it it's own PR). If y'all are ok with it I'll just sneak it in this change (nothing is listening for this event anymore so it's dead code).

File: components/d2l-quick-eval/d2l-quick-eval-submissions-table.js [hunk 2]
```diff
@@ -22,9 +22,9 @@ import {StringEndsWith} from './compatability/ie11shims.js';
  * @polymer
  */
 
-class D2LQuickEvalActivitiesList extends QuickEvalLogging(QuickEvalLocalize(PolymerElement)) {
+class D2LQuickEvalSubmissionsTable extends QuickEvalLogging(QuickEvalLocalize(PolymerElement)) {
 	static get template() {
-		const quickEvalActivitiesListTemplate = html`
+		const quickEvalSubmissionsTableTemplate = html`
```
Comment (AlexBedley) [inline components/d2l-quick-eval/d2l-quick-eval-submissions-table.js]: Ok I lied I renamed some of the variables 🤷‍♂

File: components/d2l-quick-eval/d2l-quick-eval-submissions-table.js [hunk 3]
```diff
@@ -454,7 +454,7 @@ class D2LQuickEvalActivitiesList extends QuickEvalLogging(QuickEvalLocalize(Poly
 	_dispatchLoadMore() {
 		this.dispatchEvent(
 			new CustomEvent(
-				'd2l-quick-eval-submission-table-load-more',
+				'd2l-quick-eval-submissions-table-load-more',
```
Comment (AlexBedley) [inline components/d2l-quick-eval/d2l-quick-eval-submissions-table.js]: I noticed I accidentally forgot the `s` at the end of submissions here.

File: components/d2l-quick-eval/d2l-quick-eval-submissions.js [hunk 4]
```diff
@@ -85,16 +85,16 @@ class D2LQuickEvalSubmissions extends mixinBehaviors(
...
-			<d2l-quick-eval-activities-list
+			<d2l-quick-eval-submissions-table
 				href="[[href]]"
 				token="[[token]]"
 				logging-endpoint="[[loggingEndpoint]]"
 				master-teacher="[[masterTeacher]]"
 				_data="[[_data]]"
 				_header-columns="[[_headerColumns]]"
-				on-d2l-quick-eval-submission-table-load-more="_loadMore"
+				on-d2l-quick-eval-submissions-table-load-more="_loadMore"
```
Comment (AlexBedley) [inline components/d2l-quick-eval/d2l-quick-eval-submissions.js]: This is the other half of the missing `s` change

File: components/d2l-quick-eval/d2l-quick-eval-submissions.js (hunk 1 above)
Comment (jeroach) [reply components/d2l-quick-eval/d2l-quick-eval-submissions.js]: As you said, ruthlessly delete dead code :)

**Pull request merged**